site/content/en/blog/releases/1.34.0.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "1.34.0 - Kubernetes 1.27 Support, Allocation State Filter to Stable, and all the Addresses for GameServers"
+linktitle: "1.34.0"
+date: "2023-08-15"
+---
+
+This is the 1.34.0 release of Agones.
+
+In this release, we've upgraded Kubernetes compatibility, graduated StateAllocationFilter to stable, and enhanced GameServers and GameServerAllocation address support.
+
+- Kubernetes Support: The latest update extends support to Kubernetes versions 1.25, 1.26, and 1.27, enhancing compatibility with current Kubernetes deployments. 
+
+- Allocation State Filter is now Stable: The allocation state filter feature has been promoted to stable. It determines the specific state that should be filtered when trying to obtain a `GameServer` through allocation. By default, this filter is set to `Ready`, but it can also be set to Allocated. When the `Allocated` option is selected, it can be combined with label or player selectors to access an already Allocated GameServer.
+
+- Custom Fleet Autoscaler Syncornisation Interval is now Stable: The ability to specify the interval in which the Fleet autoscaler syncs its state and updates its backing Fleet has now also been moved to Stable.
+
+- GameServer Address Improvements: GamesServers and GameServerAllocation results now also return the full set of Address records that are attached to the node they came from. This can be useful for dual stack ipv4 and ipv6 nodes, nodes with hostnames and more. 
```

Comment:
nit: If you're editing this anyways, can we use IPv4 and IPv6 (capitalized)?

(I meant to edit the doc.)

Reply:
I've updated it in release blog/draft and approved doc👍🏻